common/src/main/resources/assets/voicechat/lang/en_us.json

```diff
@@ -1,217 +1,212 @@
+  "message.voicechat.onboarding.activation.voice": "%s automatically activates your microphone when a certain volume threshold is reached.",
   "message.voicechat.onboarding.activation.voice.name": "Voice Activation",
   "message.voicechat.onboarding.ptt.title": "Choose a Push to Talk Keybind",
   "message.voicechat.onboarding.ptt.description": "Please select the key that should be used to activate your microphone.",
-  "message.voicechat.onboarding.ptt.button_description": "Press to reassign",
+  "message.voicechat.onboarding.ptt.button_description": "Press again to reassign",
   "message.voicechat.onboarding.voice.title": "Adjust Microphone Activation Threshold",
-  "message.voicechat.onboarding.voice.description": "Enable microphone testing and make sure the volume is below the slider when you're silent and above it when you are speaking.",
+  "message.voicechat.onboarding.voice.description": "Enable microphone testing and make sure the volume is below the slider when you are silent and above it when you are speaking.",
   "message.voicechat.onboarding.final": "Finish Setup",
   "message.voicechat.onboarding.final.description.success": "You have successfully configured your voice chat!\n\nOpen the voice chat menu with %s.",
-  "message.voicechat.onboarding.final.description.ptt": "Hold %s while talking to use the voice chat.",
-  "message.voicechat.onboarding.final.description.voice": "You will be muted when exiting this screen. Press %s to unmute.",
-  "message.voicechat.onboarding.final.description.configuration": "You can always change all settings in the voice chat menu.\nKeybinds can be changed like any other keybind in the game.",
-  "message.voicechat.onboarding.final.finish_setup": "Finish setup",
+  "message.voicechat.onboarding.final.description.ptt": "Hold %s while speaking to use the voice chat.",
+  "message.voicechat.onboarding.final.description.voice": "You will be muted after exiting this screen. Press %s to unmute.",
+  "message.voicechat.onboarding.final.description.configuration": "You can always change all the settings in the voice chat menu.\nKeybinds, however, can be changed just like any other keybind in the game.",
+  "message.voicechat.onboarding.final.finish_setup": "Finish Setup",
   "resourcepack.voicechat.classic_icons": "Classic Icons",
-  "resourcepack.voicechat.classic_icons.description": "Classic Voice Chat Icons",
+  "resourcepack.voicechat.classic_icons.description": "Classic voice chat icons",
   "resourcepack.voicechat.white_icons": "White Icons",
-  "resourcepack.voicechat.white_icons.description": "White Voice Chat Icons",
+  "resourcepack.voicechat.white_icons.description": "White voice chat icons",
   "resourcepack.voicechat.black_icons": "Black Icons",
-  "resourcepack.voicechat.black_icons.description": "Black Voice Chat Icons",
+  "resourcepack.voicechat.black_icons.description": "Black voice chat icons",
```

Comment:
Why lowercase?

Reply:
The reason is very simple. It is a description of a particular resource pack, and resource pack descriptions in Minecraft are usually written the way you would normally write a sentence but with no period at the end for some reason (unlike the title, which should use the title case for consistency). Mojang, LexForge, NeoForge, and many third-party resource pack authors do it this way.

![Resource-pack-description-comparison](https://github.com/user-attachments/assets/09f82caf-52bc-4ebb-9c10-a33c3c1478bc)

Reply:
Fair enough